appengine/htsget.go

```diff
@@ -6,7 +6,7 @@ import (
 	"strings"
 
 	"cloud.google.com/go/storage"
-	"github.com/googlegenomics/htsget/internal/api"
+	"github.com/googlegenomics/htsget/htsget-server/api"
```

Comment:
@anamanolache You sure `github.com/googlegenomics/htsget/htsget-server/api` is public yet?

Reply:
I see it's part of this release.  You might want to add Travis to check this works.